data/fc/scenarios/102.json

```
@@ -5,6 +5,13 @@
   "unlocks": [
     "110"
   ],
+  "requirements": [
+    {
+      "global": [
+        "Knowledge is Power:2"
```

```suggestion
        "Knowledge is Power:2"
```
I haven't checked if this works. May have to implement. Right now, requirements for many FC scenarios are not checked.

---

Should already work like this! :+1: